addons/medical_gui/stringtable.xml

```diff
@@ -844,6 +844,34 @@
             <Chinesesimp>大量失血</Chinesesimp>
             <Chinese>大量失血</Chinese>
         </Key>
+        <!--
+            Strings above and below this seem to differ in some languages, determine which is best to use
+        -->
+        <Key ID="STR_ACE_Medical_GUI_Lost_Blood1">
```

Comment:
After seeing the text strings, I would rename this one as "GUI_Lost_BloodGeneric" to avoid confusion with Blood2 message. Blood1 is "Lost a lot of blood", Blood2 is "Lost some blood".

This is a minor wish. so PR approved

Reply:
Yeah this is what I meant about them being off, I'll make a separate PR to fix these